evalai/challenges.py

```diff
@@ -264,18 +283,136 @@ def submit(ctx, file, annotation, large, public, private):
+                            while True:
+                                value = click.prompt(
+                                    style(message, fg="yellow"), type=bool, default=""
+                                )
+                                if not attribute_required or value != "":
+                                    break
+                                echo(
+                                    "Error: {} is a required field".format(
+                                        attribute_name)
+                                )
+                        if attribute_type == "radio":
+                            while True:
+                                value = click.prompt(
+                                    style(
+                                        "{}: Choices:{}".format(
+                                            message, attribute["options"]
+                                        ),
+                                        fg="yellow",
+                                    ),
+                                    type=click.Choice(attribute["options"]),
+                                    default=""
+                                )
+                                if not attribute_required or value != "":
+                                    break
+                                echo(
+                                    "Error: {} is a required field".format(
+                                        attribute_name)
+                                )
+                        if attribute_type == "checkbox":
+                            optionChosen = True
+                            while optionChosen:
+                                value = []
+                                choices = click.prompt(
+                                    style(
+                                        "{}: Choices(0 or more separated by comma):{}".format(
+                                            message, attribute["options"]
+                                        ),
+                                        fg="yellow",
+                                    ),
+                                    type=str,
+                                    show_default=False,
+                                    default=""
+                                )
+                                if choices != "":
+                                    choices = [
+                                        choice.strip(" ")
+                                        for choice in choices.split(",")
+                                    ]
+                                else:
+                                    choices = []
+                                    optionChosen = False
+                                if attribute_required and len(choices) == 0:
+                                    echo(
+                                        "Error: {} is a required field. Please select atleast one option".format(
```

Comment:
If the field is required you are showing `FieldName*` in the message it should only be `FieldName`